hnn_core/dipole.py

```diff
@@ -318,3 +329,38 @@ def write(self, fname):
                    self.data['L5']]].T
         np.savetxt(fname, X, fmt=['%3.3f', '%5.4f', '%5.4f', '%5.4f'],
                    delimiter='\t')
+
+    # Savitzky-Golay filtering, lifted and adapted from mne-python (0.22)
+    def savgol_filter(self, h_freq):
```

Comment:
you have to lift the test as well :) getting lazy? :P

Reply:
I like the idea of a Savitzky–Golay filter, but I think this code should go in the `smooth()` method. I'm late to the party here so if this has already been discussed and resolved, my suggestion isn't a big deal.